### PR TITLE
Document advice for sites with a mix of AMP & not.

### PR DESCRIPTION
**Pull request by twifkak**

### CODE REVIEW

File: docs/amp_cache_differences.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+In the meantime, web publishers must decide. If their site is primarily AMP and
+they wish to fix their URLs, use an AMP tool. If their site is primarily
+non-AMP and they wish to speed up page loads with prefetching, use a general
+tool.
+
+For sites with a mix of AMP and non-AMP, it may be acceptable to run a general
+tool to speed up page loads. The only downside is the presence of [Search
+Console
+warnings](https://support.google.com/webmasters/answer/7450883#sgx_warning_list)
+that indicate that AMP is being treated as typical, instead of as a SXG.
```
Comment (banaag) [inline docs/amp_cache_differences.md:32]: a SXG -> an SXG?

File: docs/amp_cache_differences.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Advice for sites with a mix of AMP and non-AMP
+
+The Google SXG Cache requirements differ from the earlier [Google AMP Cache
+requirements](https://github.com/ampproject/amppackager/blob/releases/docs/cache_requirements.md)
+for signed exchanges.
+
+The original reasoning for the latter was to validate that AMP SXGs were at
+parity with the stronger constraints of AMP. Today, if an AMP page is signed
```
Comment (banaag) [inline docs/amp_cache_differences.md:8]: signed -> signed after

File: docs/amp_cache_differences.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+ - Run [AMP
+   Optimizer](https://github.com/ampproject/amp-toolbox/tree/master/packages/optimizer)
+   on AMP pages in order to meet the transformed requirement.
+ - Modify AMP Optimizer's output to add the [`data-sxg-no-header`
+   attribute](https://github.com/google/sxg-rs/blob/main/README.md#preload-subresources)
+   to any `<link rel=preload>` tags. This will prevent Cloudflare Automatic
+   Signed Exchanges from turning them into Link headers that wouldn't meet the
+   AMP SXG requirements.
+ - Other changes may be required.
```
Comment (banaag) [inline docs/amp_cache_differences.md:45]: Could you provide examples of these changes?

File: docs/amp_cache_differences.md (hunk 2 above)
Comment (twifkak) [reply docs/amp_cache_differences.md]: signed meeting -> signed, but it meets

File: docs/amp_cache_differences.md (hunk 3 above)
Comment (twifkak) [reply docs/amp_cache_differences.md]: Added `max-age`. And clarified that this is a catch-all for "dunno".

**Pull request merged**